src/lib/entities/audio/model/audio-instance.ts

```diff
@@ -11,8 +11,18 @@ export const paused = writable(true);
 export const buffered = writable<SvelteTimeRanges>([]);
 
 export const play = (_src?: string) => {
-  if (_src) src.set(_src);
-  paused.set(false);
+  if (_src) {
+    pause();
+    duration.set(0);
+    let s: Unsubscriber | undefined = undefined;
+    s = duration.subscribe((v) => {
```

Comment:
I don't think it needs to have `undefined` in its type since it never gets assigned to it. This can simply become
```suggestion
    const s = duration.subscribe((v) => {
```
and the optional chaining won't be needed

Reply:
Also, a more meaningful name (such as `unsubscribe`) would be helpful